libs/blocks/chart/utils.js

```diff
@@ -34,20 +34,12 @@ export function formatExcelDate(date) {
   let newDate;
 
   if (!Number.isNaN(+date)) {
-    const hours = Math.floor((+date % 1) * 24);
-    const minutes = Math.floor((((+date % 1) * 24) - hours) * 60);
-    const offsetUTC = 24 - (new Date().getTimezoneOffset() / 60);
-
-    newDate = new Date(Date.UTC(0, 0, +date, hours - offsetUTC, minutes));
+    newDate = +date > 99999
+      ? new Date(+date * 1000)
+      : new Date(Math.round((+date - (1 + 25567 + 1)) * 86400 * 1000));
   } else {
     newDate = new Date(date);
   }
 
-  const localDateFormat = new Date(
-    newDate.getFullYear(),
-    newDate.getMonth(),
-    newDate.getDate(),
-  );
-
-  return localDateFormat.toLocaleString([], { dateStyle: 'short' });
+  return newDate.toLocaleString([], { dateStyle: 'short', timeZone: 'GMT' });
```

Comment:
@Brandon32 does this really fix the issue at hand? Locally this fails in GMT+2 (CEST/Zurich Time)

It converts the Date (8/5/2022) to `newDate: '2022-08-04T22:00:00.000Z'` and hence appears a day earlier. I would imagine on a real chart it would have the same issue

```
test/blocks/chart/chart.test.js:

 🚧 Browser logs:
      { date: '8/5/2022', newDate: '2022-08-04T22:00:00.000Z' }
      { date: '8/6/2022', newDate: '2022-08-05T22:00:00.000Z' }

 ❌ chart > chart mark series data
      AssertionError: expected [ Array(1) ] to deeply equal [ Array(1) ]
      + expected - actual
      
       [
         [
           {
             "name": "Weekend Sale"
      -      "xAxis": "8/4/22"
      +      "xAxis": "8/5/22"
           }
           {
      -      "xAxis": "8/5/22"
      +      "xAxis": "8/6/22"
           }
         ]
       ]
      
      at n.<anonymous> (test/blocks/chart/chart.test.js:255:39)

Chromium: |██████████████████████████████| 1/1 test files | 0 passed, 1 failed
```